Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -260,6 +270,17 @@ class MainMenuActionHelper: PhotonActionSheetProtocol, FeatureFlaggable, CanRemo
 
     // MARK: - Actions
 
+    private func getInviteFriendsAction() -> PhotonRowActions {
+        SingleActionViewModel(
+            title: .localized(.inviteFriends),
+            iconString: "inviteFriends",
+            isNew: User.shared.referralsEntryPointIsNew) { [weak self] action in
+                action.isNew = false
```

Comment:
Probably we also need an analytics event, don't we?

Reply:
Not in the scope. But I encourage you @ecotopian to discuss it with the team and add it to the ticket.